enterprise/models.py

```diff
@@ -197,6 +197,24 @@ class Meta:
         )
     )
 
+    TYPE_ENTERPRISE = 'enterprise'
+    TYPE_PARTNER = 'partner'
+    TYPE_NON_PROD = 'non_prod'
+    CUSTOMER_TYPE_CHOICES = (
+        (TYPE_ENTERPRISE, 'Enterprise Customer'),
+        (TYPE_PARTNER, 'Partner Customer'),
+        (TYPE_NON_PROD, 'Non Production Customer'),
```

Comment:
The label `Customer type` is indicating the type of customers.  I think, We should eliminate the word `Customer` from the options. 

Customer Types: 
1. Enterprise
2. Partners
3. Non Production